rest-api-spec/src/main/resources/rest-api-spec/test/indices.shrink/40_max_shard_size.yml

```diff
@@ -4,8 +4,8 @@
   # number_of_shards for the target index.
 
   - skip:
-      version: " - 2.99.99"
-      reason: "only available in 3.0+"
+      version: " - 2.4.99"
```

Comment:
I'm not very familiar with this area, so this might be a naïve question; is there a test we can add that will verify this behavior?

Reply:
I think if all the yml rest tests have passed it means this change works, especially if `MixedClusterClientYamlTestSuiteIT` has passed it means this test works for both 3.0.0 and 2.12.0 version. The supported version is [2.5](https://github.com/opensearch-project/OpenSearch/blob/2.x/rest-api-spec/src/main/resources/rest-api-spec/test/indices.shrink/40_max_shard_size.yml) in 2.x branch. Why we set the supported version to 3.0.0 in the yml file of the main branch before is that the  `MixedClusterClientYamlTestSuiteIT` could be failed definitely if the original PR had not been backported to 2.x branch, after the backport is done, we can change the supported version to 2.5 in main branch, though it's too late to do that.